src/wp-admin/includes/ajax-actions.php

```diff
@@ -158,6 +158,7 @@ function wp_ajax_ajax_tag_search() {
 			'name__like' => $s,
 			'fields'     => 'names',
 			'hide_empty' => false,
+			'number'     => isset( $_GET['number'] ) ? (int) $_GET['number'] : 0,
```

Comment:
WP_Term_Query::parse_query() performs `absint( $query['number'] )` [docs](https://developer.wordpress.org/reference/functions/absint/), so the `(int)` cast isn't needed.

Reply:
@costdev @peterwilsoncc so are we ok to ship this PR?